run_YACHT.py

```diff
@@ -57,8 +60,8 @@
     ani_thresh = config['ani_thresh']
 
     # Make sure the output can be written to
-    if not os.access(os.path.abspath(os.path.dirname(out_filename)), os.W_OK):
-        raise FileNotFoundError(f"Cannot write to the location: {os.path.abspath(os.path.dirname(out_filename))}.")
+    if not os.access(outdir, os.W_OK):
```

Comment:
If the `outdir` is relatively references (eg. `.`) then the `os.access` may not see that it's actually writable. I came upon the `os.path.abspath(os.path.dirname(out_filename))` construct by experimenting with `out_filename`'s of the form `out_file.ext`, `/abs/path/to/out_file.ext`, `../../rel_path/out_file.ext` etc.

Reply:
Please see my above comment. The `outdir` works for all different formats.

Reply:
Please revert this: having two arguments (location and name) for one thing (the output) is not good style